docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,133 @@
+```
+
+
+## Install Kyma
+
+There are several ways to install Kyma:
+
+- To install Kyma on a local cluster, you can simply use the `deploy` command.<br>
+Kyma provides a default domain under the URL `https://console.local.kyma.dev`.
+
+    ```bash
+    kyma alpha deploy 
+    ```
+   
+- You can also install Kyma using your own domain name, or on a cluster that does not run locally.<br>
+    If you use a custom domain, you must provide the certificate and key as files. If you don't have a certificate yet, you can create a self-signed certificate and key:
+
+    ```bash
+    openssl req -x509 -newkey rsa:4096 -keyout key.pem -out crt.pem -days 365
+    ```
+    When prompted, provide your credentials, such as your name and your domain.
+
+    Then, pass the certificate files to the deploy command:
+
+        ```bash
+        kyma alpha deploy --domain {DOMAIN} --tls-cert crt.pem --tls-key key.pem
+        ```
+
+- Optionally, you can specify from which source you want to deploy Kyma, such as the `master` branch, a specific PR, or a release version. For more details, see the documentation for the `alpha deploy` command.<br>
+For example, to install Kyma from a specific version, such as `1.19.1`, run:
+
+    ```bash
+    kyma alpha deploy --source=1.19.1
+    ```
+
+- Alternatively, to build Kyma from your local sources and deploy it on a remote cluster, run:
+
+    ```bash
+    kyma alpha deploy --source=local
+    ```
+    > **NOTE:** By default, Kyma expects to find the sources in the `./workspace folder`. To adjust the path, set the `-w ${PATH_TO_KYMA_SOURCES}` parameter.
+
+- To deploy Kyma with only specific components, run:
+
+    ```bash
+    kyma alpha deploy --components {COMPONENTS_FILE_PATH}
+    ```
+    `{COMPONENTS_FILE_PATH}` is the path to a YAML file containing the desired component list to be installed. In the following example, only these 6 components are deployed on the cluster:
+
+    ```bash
+    defaultNamespace: kyma-system
+    prerequisites:
+      - name: "cluster-essentials"
+      - name: "istio"
+        namespace: "istio-system"
+    components:
+      - name: "testing"
+      - name: "xip-patch"
+      - name: "istio-kyma-patch"
+      - name: "dex"
+    ```
+
+- To override the standard Kyma installation, run:
+
+    ```bash
+    kyma alpha deploy --values-file {OVERRIDE_FILE_PATH}
+    ```
+    `{OVERRIDE_FILE_PATH}` is the path to a YAML file containing the parameters to override.<br>
+In the following example, overrides are provided for 2 different components: `ory` and `monitoring`.
+ 
+    - For `ory`, the values of `hydra.deployment.resources.limits.cpu` and `hydra.deployment.resources.requests.cpu` will be overriden to `153m` and `53m` respectively.
+    - For `monitoring`, the values of `alertmanager.alertmanagerSpec.resources.limits.memory` and `alertmanager.alertmanagerSpec.resources.requests.memory` will be overriden to `304Mi` and `204Mi` respectively.
+    ```bash
+     ory:
+      hydra:
```

Comment:
Same problem as above on the website preview :
<img width="569" alt="Screen Shot 2021-02-22 at 08 43 22" src="https://user-images.githubusercontent.com/49867570/108677555-2103d280-74ea-11eb-92cb-eff44acf06f7.png">

```suggestion
    - For `ory`, the values of `hydra.deployment.resources.limits.cpu` and `hydra.deployment.resources.requests.cpu` will be overriden to `153m` and `53m` respectively.
    
    - For `monitoring`, the values of `alertmanager.alertmanagerSpec.resources.limits.memory` and `alertmanager.alertmanagerSpec.resources.requests.memory` will be overriden to `304Mi` and `204Mi` respectively.
    
    ```bash
     ory:
      hydra:
```